src/features/components/molecules/trade-item/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { trade_item } from './style.module.scss'
+import { Localize } from 'components/localization'
+import Image from 'features/components/atoms/image'
+import Typography from 'features/components/atoms/typography'
+import Flex from 'features/components/atoms/flex-box'
+import { FlexBoxProps } from 'features/components/atoms/flex-box/box'
+import { TString } from 'types/generics'
+
+export type TradeItemType = {
+    icon_src: string
+    heading: TString
+    description: TString
+}
+
+type TradeItemProps = FlexBoxProps<'div'> & {
+    data: TradeItemType
+}
+
+const TradeItem = ({ data, ...rest }: TradeItemProps) => {
+    return (
+        <Flex.Box
+            direction="col"
+            align="center"
+            grow="1"
+            md={{ basis: '3-12' }}
+            className={trade_item}
```

Comment:
Could you please check? I believe it's the same
```suggestion
            basis= {'3-12'}
```

Reply:
No, it's not same. We need 3 columns in desktop and 1 column in mobile here. That will make 3 columns for both mobile and desktop.